lib/bolt_spec/plans.rb

```diff
@@ -202,7 +202,7 @@ def run_plan(name, params)
     end
 
     def allow_apply_prep
-      allow_task('apply_helpers::custom_facts')
+      allow_task('apply_helpers::custom_facts').be_called_times(100)
```

Comment:
Why the 100 limit?

Reply:
It was a hack, 100 seemed unlikely to be reached for apply_prep or get_resources in a plan.

Reply:
Probably needs to be reworked to not enforce a limit unless explicitly specified. Which might not be hard now that I think about it.